src/Model/Page.php

```diff
@@ -582,6 +589,22 @@ public function setTarget(PageInterface $target = null): void
         $this->target = $target;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getTargetRedirectType(): int
+    {
+        return $this->targetRedirectType;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function setTargetRedirectType(int $statusCode = Response::HTTP_FOUND): void
```

Comment:
Shoudn't it be `?int` becuase there is no point in ` = Response::HTTP_FOUND`

Reply:
wouldn't that mean that you explicitly can set it to `null`? I guess the auto assign doesn't make sense then.

Reply:
https://3v4l.org/PK4rQ you are correct, my bad